tests/test_cluster_utils.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import pytest
+
+from ocsci.testlib import tier1, ManageTest
+from ocs.cluster import CephCluster
+
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    Create disks
+    """
+    self = request.node.cls
+
+    def finalizer():
+        teardown()
+    request.addfinalizer(finalizer)
+    setup(self)
+
+
+def setup(self):
+    """
+    Setting up the environment for the test
+    """
+    global cluster
+    cluster = CephCluster()
+
+
+def teardown():
+    """
+    Tearing down the environment
+    """
+    pass
+
+
+@tier1
+@pytest.mark.usefixtures(
+    test_fixture.__name__,
+)
+class TestClusterUtils(ManageTest):
+
+    username = "client.test"
+    caps = "mon 'allow r' osd 'allow rwx'"
+
+    def test_user_creation(self):
```

Comment:
I'm not sure this is a valid test for OCS.
It is not harmful to have it but I'm not sure it will have any value

Reply:
Is the created user deleted somewhere? Should be in finalizer but we probably don't want to have such tests of your functionality as regular tests as I mentioned but as unittests.

Reply:
Done, added setup and teardown which takes care of user creation and other resource deletion